core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -81,7 +81,7 @@ private CarbonCommonConstants() {
    * default carbon properties file path
    */
   public static final String CARBON_PROPERTIES_FILE_PATH_DEFAULT =
-      "../../../conf/carbon.properties";
```

Comment:
please keep the old default value. if we change it, it will impact the user to upgrade the old version to a new version.

Reply:
hi, I got your point. But may be most user don't use this defalut value, because `../../../` means the property file was placed at three-tier parent directory.
If you still stick to your point of view, I am also willing to revert it, thanks.

Reply:
I think it is better to keep default value otherwise it will break compatibility when user upgrade to new version

Reply:
> I think it is better to keep default value otherwise it will break compatibility when user upgrade to new version

ok, I see. I'll revert it.